app/mailers/precheck_mailer.rb

```diff
@@ -7,7 +7,7 @@ def precheck_completed(family)
             else
               DEFAULT_EMAIL
             end
-    mail(to: email, subject: "#{UserVariable[:CONFID]} - Precheck Completed")
+    mail(to: email, subject: "#{UserVariable[:conference_id]} - Precheck Completed")
```

Comment:
I changed the keys because it seems like this is how the other UserVariables are setup (in the seeder). Honestly I'm not sure why the data model has both a short_name and also a code, so for now I just tried to copy how it was done previously.

Reply:
Agreed, its very confusing the whole user variable naming options. 
The other user variables I saw were all capitalized and short, so thats why I went that route. I didn't take a decision at the spot and kept 'going with it', but I believe these more expressive names are better anyways, so fully support this. 🥇